.devcontainer/noop.txt

```diff
@@ -0,0 +1,3 @@
+This file copied into the container along with environment.yml* from the parent
+folder. This file is included to prevents the Dockerfile COPY instruction from 
+failing if no environment.yml is found.
```

Comment:
Suggest small change:
This file is copied into the container along with environment.yml* from the parent
folder.  It is included to prevent the Dockerfile COPY instruction from 
failing if no environment.yml is found.

Reply:
Done.